.github/workflows/tests.yml

```diff
@@ -0,0 +1,352 @@
+name: 'Tests'
+
+on:
+  pull_request:
+    paths-ignore:
+      - 'docs/**'
+
+jobs:
+  lint:
+    name: 'lint (node: ${{ matrix.node }})'
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node: [12, 14]
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2-beta
+        with:
+          node-version: ${{ matrix.node }}
+      - name: Install modules
+        run: yarn
+      - name: Run lint
+        run: yarn run -s lint
+
+  unit_back:
+    name: 'unit_back (node: ${{ matrix.node }})'
+    needs: [lint]
+    runs-on: ubuntu-latest
+    env:
+      CODECOV_TOKEN: ${{ secrets.codecov }}
+    strategy:
+      matrix:
+        node: [12, 14]
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2-beta
+        with:
+          node-version: ${{ matrix.node }}
+      - name: Install modules
+        run: |
+          yarn && yarn global add codecov
+          echo "$(yarn global bin)" >> $GITHUB_PATH
+      - name: Run test
+        run: yarn run -s test:unit --coverage && codecov -C -F unit
+
+  unit_front:
+    name: 'unit_front (node: ${{ matrix.node }})'
+    needs: [lint]
+    runs-on: ubuntu-latest
+    env:
+      CODECOV_TOKEN: ${{ secrets.codecov }}
+    strategy:
+      matrix:
+        node: [12, 14]
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2-beta
+        with:
+          node-version: ${{ matrix.node }}
+      - name: Install modules
+        run: |
+          yarn && yarn global add codecov
+          echo "$(yarn global bin)" >> $GITHUB_PATH
+      - name: Run test
+        run: yarn run -s test:unit && codecov -C -F front
+
+  e2e_ce_pg:
+    runs-on: ubuntu-latest
+    needs: [lint, unit_back, unit_front]
+    name: 'E2E (pg:${{ matrix.postgres }} node: ${{ matrix.node }})'
```

Comment:
Could you prefix the name with `[CE]`  to match EE tests? (eg: `name: '[EE] E2E (mysql node: ${{ matrix.node }})'`)
(same for other CE jobs)

Reply:
Also, could you add some kind of separator between db type & node version? It would be a lot easier to read :slightly_smiling_face: